canary/webrtc-c/src/Peer.cpp

```diff
@@ -214,13 +241,31 @@ STATUS Peer::initRtcConfiguration(const Canary::PConfig pConfig)
     }
 
     if (pConfig->useTurn.value) {
-
+        // Set the URIs from the configuration
+        CHK_STATUS(awaitGetIceConfigInfoCount(signalingClientHandle, &iceConfigCount));
         /* signalingClientGetIceConfigInfoCount can return more than one turn server. Use only one to optimize
          * candidate gathering latency. But user can also choose to use more than 1 turn server. */
         for (uriCount = 0, i = 0; i < MAX_TURN_SERVERS; i++) {
             CHK_STATUS(signalingClientGetIceConfigInfo(signalingClientHandle, i, &pIceConfigInfo));
-            CHECK(uriCount < MAX_ICE_SERVERS_COUNT);
-            uriCount += pIceConfigInfo->uriCount;
+            for (j = 0; j < pIceConfigInfo->uriCount; j++) {
+                CHECK(uriCount < MAX_ICE_SERVERS_COUNT);
```

Comment:
Not sure where CHECK is defined but this seems redundant if max count is always 1. Are we planning on adding the different turn server Urls (turn with tcp, or turns w/ udp)

Reply:
CHECK is defined in PIC PlatformUtils. Not planning on adding more turn server urls, but leaving in case we ever do and to replicate the same way we do this in the samples [here](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/30f52d18e647747de1ec29ad6f6d3f62128f5494/samples/Common.c#L383) with maxTurnServer set to 1 [here](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/30f52d18e647747de1ec29ad6f6d3f62128f5494/samples/Common.c#L353).